sentences/de/weather_HassGetWeather.yaml

```
@@ -0,0 +1,10 @@
+language: de
+intents:
+  HassGetWeather:
+    data:
+      - sentences:
+          - "wie ist [das] Wetter [(für|in) <name>]"
+          - "wie ist [das] <name> Wetter"
```

```suggestion
          - "wie ist [das] <name>[er] Wetter"
```

what do you think about this addition? it should make "Wie ist das Berliner Wetter?" possible 🤔

---

I like that idea! 😃

---

I also added your test sentence